taipy/gui/viselements.json

```diff
@@ -1599,6 +1570,40 @@
                                 "dict"
                             ]
                         ]
+                    },
+                    {
+                        "name": "selected",
+                        "type": "dynamic(Union[str,list[str]])",
+                        "doc": "Semicolon (';')-separated list or a list of menu items identifiers that are selected."
```

Comment:
"type": "dynamic(list[str])",
"doc": "Semicolon (';')-separated list or a list of menu items identifiers that will be selected."

Reply:
updated